packages/flutter_rx_bloc/pubspec.yaml

```diff
@@ -11,18 +11,18 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  provider: ^6.0.5
+  provider: ^6.1.1
```

Comment:
Do we really need to upgrade to 6.1.1?

Reply:
We don't, but it's the latest version

Reply:
I guess no.
I have reverted all the dependencies upgrades